cni/pkg/plugin/plugin.go

```diff
@@ -284,7 +284,7 @@ func doRun(args *skel.CmdArgs, conf *Config) error {
 	}
 
 	if pi.ProxyType != "" && pi.ProxyType != "sidecar" {
-		log.Infof("excluded because it has proxy type %v", podNamespace, podName, pi.ProxyType)
+		log.Infof("excluded %s/%s because it has proxy type %s", podNamespace, podName, pi.ProxyType)
```

Comment:
```suggestion
		log.Infof("excluded pod %s/%s because it has proxy type %s", podNamespace, podName, pi.ProxyType)
```

Reply:
Thank you. Updated. 
I changed `excluded %s/%s pod` instead of `excluded pod %s/%s` because other log lines like L264 has `Failed to get %s/%s pod`.